src/ping.ts

```diff
@@ -1,8 +1,9 @@
 'use strict';
 
 import { Client } from './client';
+import { NamedAgent } from './types/named-agent';
 
-const DEFAULT_PING_INTERVAL = 15 * 1000; // same default as stanza message timeouts
+const DEFAULT_PING_INTERVAL = 14 * 1000;
```

Comment:
Any reason you changed this to 14 from 15? Did it change in stanza?

Reply:
Yeah, I changed it because *sometimes* the stanza timeout would trigger first and the logs wouldn't indicate a reason why it was disconnected.